lib/modules/manager/flux/extract.ts

```diff
@@ -120,26 +124,29 @@ function resolveSystemManifest(
 function resolveResourceManifest(
   manifest: ResourceFluxManifest,
   helmRepositories: HelmRepository[],
+  helmCharts: HelmChart[],
   registryAliases: Record<string, string> | undefined,
 ): PackageDependency[] {
   const deps: PackageDependency[] = [];
   for (const resource of manifest.resources) {
     switch (resource.kind) {
       case 'HelmRelease': {
-        const dep: PackageDependency = {
-          depName: resource.spec.chart.spec.chart,
-          currentValue: resource.spec.chart.spec.version,
-          datasource: HelmDatasource.id,
-        };
+        if (resource.spec.chartRef?.kind === 'OCIRepository') {
+          // Can be skipped, version is handled via OCIRepository resource directly
+          break;
```

Comment:
why break instead of continue?
```suggestion
          continue;
```

log a trace message